config/plugins/visualizations/jupyterlite/config/jupyterlite.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is going to be weird for notebooks that use input datasets, i.e. those that were run by the jupyter IT / jupytool. I guess there's not much we can do ?

Reply:
We could maybe create a different datatype? But I guess at some point users need to understand what the limits of the `lite` versions are.